application/app/components/fiches/ficheCommune.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+
+import { CommuneProperties } from '@/app/models/communes';
+
+import AccordionCard from './shared/AccordionCard';
+import ActionButtons from './shared/ActionButtons';
+import CollectiviteHeaderCard from './shared/CollectiviteHeaderCard';
+import PotentielSolaireCard from './shared/PotentielSolaireCard';
+import RepartitionPotentielSolaire from './shared/RepartitionPotentielSolaire';
+import ResponsabiliteMessage from './shared/ResponsabiliteMessage';
+import Tabs from './shared/Tabs';
+import TopCard from './shared/TopCard';
+
+interface FicheCommuneProps {
+	commune: CommuneProperties;
+}
+
+export default function FicheCommune({ commune }: FicheCommuneProps) {
+	const [activeTab, setActiveTab] = useState('all');
+
+	const handleTabChange = (tab: string) => {
+		setActiveTab(tab);
+	};
+
+	const tabs = [
+		{ id: 'all', label: 'Tous' },
+		{ id: 'managed', label: 'Écoles primaires' },
+	];
```

Comment:
Tu peux ajouter un type plus specifique pour eviter les erreurs et sortir la const du composant vu qu elle n a pas besoin d etre redefini a chaque render
```suggestion
const tabs = [
	{ id: 'all', label: 'Tous' },
	{ id: 'managed', label: 'Écoles primaires' },
];

type TabId = (typeof tabs)[number]['id'];

interface FicheCommuneProps {
	commune: CommuneProperties;
}

export default function FicheCommune({ commune }: FicheCommuneProps) {
	const [activeTab, setActiveTab] = useState<TabId>('all');

	const handleTabChange = (tab: TabId) => {
		setActiveTab(tab);
	};
```